src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,85 @@
-export const Pagination = () => {};
+import classNames from 'classnames';
+import { useState } from 'react';
+import { getNumbers } from '../../utils';
+
+type Props = {
+  total: number;
+  perPage: number;
+  currentPage: number;
+  onPageChange: React.MouseEventHandler<HTMLAnchorElement>;
+};
+
+export const Pagination: React.FC<Props> = ({
+  total,
+  perPage,
+  currentPage,
+  onPageChange,
+}) => {
+  const [selectedPage, setSelectedPage] = useState(currentPage);
+  const countPagination = Math.ceil(total / perPage);
+  const countMassive = getNumbers(1, countPagination);
+
+  return (
+    <ul className="pagination">
+      <li
+        id="prev"
+        className={classNames(
+          'page-item', { disabled: selectedPage === 1 },
```

Comment:
create a variable `firstPage` and use it

Reply:
Okay